examples/requirements.txt

```diff
@@ -3,12 +3,12 @@ dm_env
 dm_meltingpot
 dm_tree
 dmlab2d
-gym
+gymnasium
 matplotlib
 ml_collections
 numpy<1.23  # Needed by Ray because it uses `np.bool`.
```

Comment:
Can we unpin and uncomment this now ray is updated?

Reply:
I think so. I checked and 2.5.0 does not use np.bool, while 2.0.0 does. Although both use np.bool_, it is not deprecated.